cli.js

```diff
@@ -41,7 +42,13 @@ function open(name) {
 	if (cli.flags.github) {
 		return packageJson(name, {fullMetadata: true}).then(pkg => {
 			if (pkg.repository) {
-				const url = githubUrlFromGit(pkg.repository.url);
+				const url = githubUrlFromGit(pkg.repository.url) || pkg.repository.url;
+
+				if (!urlRegex().test(url)) {
+					console.error('Invalid repository URL, opening homepage');
```

Comment:
The message should be explicit about it being the `repository` property in the package.json being invalid and recommending the user to open an issue/PR on the repo to fix the package.json

Reply:
Changed :)